src/OpenTelemetry.Exporter.Console/ConsoleMetricExporter.cs

```diff
@@ -97,15 +97,14 @@ public override ExportResult Export(in Batch<Metric> batch)
                         var bucketsBuilder = new StringBuilder();
                         var sum = metricPoint.GetHistogramSum();
                         var count = metricPoint.GetHistogramCount();
+                        bucketsBuilder.Append($"Sum: {sum} Count: {count} ");
                         if (metricPoint.HasMinMax())
                         {
-                            bucketsBuilder.Append($"Sum: {sum} Count: {count} Min: {metricPoint.GetHistogramMin()} Max: {metricPoint.GetHistogramMax()} \n");
-                        }
-                        else
-                        {
-                            bucketsBuilder.Append($"Sum: {sum} Count: {count} \n");
+                            bucketsBuilder.Append($"Min: {metricPoint.GetHistogramMin()} Max: {metricPoint.GetHistogramMax()} ");
                         }
 
+                        bucketsBuilder.Append("\n\r");
```

Comment:
This looks very strange to me, normally folks would use `\r\n` (CRLF) on Windows.

Reply:
Curious, is the intention to use https://learn.microsoft.com/dotnet/api/system.environment.newline?